py3-cattle.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: py3-cattle
+  version: "0.5.4"
+  epoch: 0
+  description: Command line client for cattle
+  copyright:
+    - license: MIT
+  dependencies:
+    provides:
+      - cattle=${{package.full-version}}
+      - cattle-cli=${{package.full-version}}
+      - rancher-cattle-cli=${{package.full-version}}
```

Comment:
This is an empty package and should never be used. I'd remove these and set provider priority to 0 for this package explicitly